components/custom-radio-button/css/_mixin.scss

```diff
@@ -0,0 +1,106 @@
+/**
+ * @license EUPL-1.2
+ * Copyright (c) 2021 Robbert Broersma
+ */
+
+@import "../../radio-button/css";
+@import "../../common/focus/mixin";
+
+@mixin utrecht-custom-radio-button {
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  appearance: none;
+  background-color: var(--utrecht-custom-radio-button-background-color, #fff);
```

Comment:
I prefer no hardcoded default value such as `#fff`. Changing it would be a breaking change, and I'd rather force folks to explicitly set this design token to white in their JSON files if they need it to be white.